### PR TITLE
License header job fix: ignore text files, update format

### LINKED ISSUE #15067: `Fix License Headers` job on CI is failing
## Description

Since #15054 the `Fix License Headers` job is failing

## Reproduce

See https://github.com/jupyterlab/jupyterlab/actions/runs/6060071826/job/16443852957

## Expected behavior

It does not fail

## Context

None

### PR DESCRIPTION
**Pull request by krassowski**
## References

Closes #15067

## Code changes

Ignores text files in galata snapshots (`txt` is no longer automatically ignored since https://github.com/apache/skywalking-eyes/pull/141).

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

File: galata/extension/src/global.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
```
Comment (krassowski) [inline galata/extension/src/global.ts:5]: This one has undated Bloomberg copyright. Not sure what to do here.

**Comment by krassowski** [comment]
I will force-push to see if https://github.com/jupyterlab/jupyterlab/pull/15072 helped.

File: galata/extension/src/global.ts (hunk 1 above)
Comment (krassowski) [reply galata/extension/src/global.ts:7]: Well, moving the old Bloomberg copyright one line down pleases the header linter.

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**